tests/utils/PolygonUtilsTest.cpp

```diff
@@ -111,31 +115,37 @@ TEST_F(MoveInsideTest, middleTest)
     const Point result = PolygonUtils::moveInside(cpp, distance);
 
     constexpr coord_t maximum_error = 10;
-    ASSERT_TRUE(vSize(result - supposed1) <= maximum_error || vSize(result - supposed2) <= maximum_error || vSize(result - supposed3) <= maximum_error || vSize(result - supposed4) <= maximum_error)
-        << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << ", " << supposed2 << ", " << supposed3 << " or " << supposed4 << ".\n"
-        << "\tPS: dist to boundary computed = " << vSize(cpp.location - result) << "; vs supposed = " << vSize(cpp.location - supposed1) << ", " << vSize(cpp.location - supposed2) << ", " << vSize(cpp.location - supposed3) << " or " << vSize(cpp.location - supposed4) << ".\n"
-        << "\tclosest point = " << cpp.location << " at index " << cpp.point_idx << ".";
+    ASSERT_TRUE(vSize(result - supposed1) <= maximum_error || vSize(result - supposed2) <= maximum_error
+                || vSize(result - supposed3) <= maximum_error || vSize(result - supposed4) <= maximum_error)
+      << close_to << " moved with " << distance << " micron inside to " << result << " rather than " << supposed1 << ", " << supposed2
+      << ", " << supposed3 << " or " << supposed4 << ".\n"
+      << "\tPS: dist to boundary computed = " << vSize(cpp.location - result) << "; vs supposed = " << vSize(cpp.location - supposed1)
+      << ", " << vSize(cpp.location - supposed2) << ", " << vSize(cpp.location - supposed3) << " or " << vSize(cpp.location - supposed4)
+      << ".\n"
```

Comment:
This used to be formatted such that a line in the code corresponds to a line in the output. Breaking up the lines this way breaks that.